### DISCUSSION TITLE
To observe some interesting map issues you should add Add/Remove/Clear/Replace operations to MauiMaps

### DISCUSSION THREAD

**Discussion by davefxy** [VladislavAntonyuk/MauiSamples#111] [Q&A] [answered]
Combine your MauiMaps and dotnet\maui\src\Controls\samples\Controls.Sample\Pages\Controls\MapsGalleries\PinItemsSourcePage ()Add/Remove/Clear/Replace operations). You will see some interesting results displayed by the app. For example, when deleting an annotated pin entry, the default icon gets deleted, but the annotated image remains. Map doesn't always reflect the map pin updates immediately.
If you would be so kind as to point me to info on how to setup the dotnet\maui repository so I can debug some of the issues I occasionally come across. I have tried following the development.md. But I cant get the resulting local bin\dotnet to compile successfully. I will understand if you haven't the time.

**Comment by VladislavAntonyuk** [comment]
The only one thing you need is install dotnet 7 and install MAUI workload. VS preview installer does it for you. 
Android maps requires storing Markers, so you can access them later. I will update app. You should not see such issue on iOS

**Comment by VladislavAntonyuk** [comment, marked as answer]
Fixed by https://github.com/VladislavAntonyuk/MauiSamples/pull/116
